common/src/main/scala/org/apache/celeborn/common/meta/AppDiskUsageMetric.scala

```diff
@@ -143,13 +143,8 @@ class AppDiskUsageMetric(conf: CelebornConf) extends Logging {
           currentSnapShot.get().commit()
         }
         currentSnapShot.set(getNewSnapShot())
-        val summaryStr = Some(summary()).map(str =>
-          if (str != null && str.nonEmpty) {
-            "\n" + str
-          } else {
-            str
-          }).getOrElse("")
```

Comment:
These checks were not needed as we are already treating summary as scala option.